addons/l10n_fr/data/tax_report_data.xml

```diff
@@ -256,6 +256,15 @@
         <field name="sequence">15</field>
     </record>
 
+    <record id="tax_report_F9" model="account.tax.report.line">
+        <field name="name">F9 - Opérations internes réalisées entre membres d'un assujetti unique</field>
+        <field name="tag_name">F9</field>
+        <field name="code">box_F9</field>
+        <field name="parent_id" ref="tax_report_op_non_imposables"/>
+        <field name="report_id" ref="tax_report"/>
+        <field name="sequence">16</field>
```

Comment:
This is perfectly fine in 14.0, but when forward-porting to 16.0 don't forget the sequences become global: they're not done by parent like on account.tax.report.line :)